pkgs/development/libraries/ucx/default.nix

```
@@ -1,7 +1,8 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen
-, numactl, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, doxygen, numactl
+, rdma-core, libbfd, libiberty, perl, zlib, symlinkJoin, pkgconfig
+, cudatoolkit, rocm-core, rocm-runtime, rocm-device-libs, hip
 , enableCuda ? false
```

The optional switches should be grouped with their dependencies like this:
```suggestion
, enableCuda ? false
, cudatoolkit
, enableRocm ? false
, rocm-core, rocm-runtime, rocm-device-libs, hip
```